src/shared/components/annotation/CancerHotspots.tsx

```diff
@@ -122,7 +120,7 @@ export default class CancerHotspots extends React.Component<ICancerHotspotsProps
         const {isHotspot, is3dHotspot} = this.props;
 
         let hotspotContent = (
-            <span/>
+            <span className={`${annotationStyles["annotation-item"]}`} />
```

Comment:
This line is duplicated in `OncoKB` and `MyCancerGenome` components as well. It might be better to add an `AnnotationItemPlaceholder` component for this one. Not sure if it is a good idea to create a component for something this small though.